src/tasks.js

```diff
@@ -0,0 +1,23 @@
+export const tasks = [
+  {
+    description: 'Go to the market',
+    completed: false,
+    index: 0,
+  },
+];
```

Comment:
- [ ] Please 🙏🏻 adhere to the project requirements, according to which your `task` array should render on the UI as soon as your page loads, but I noticed that you are not rendering the task array anywhere and hence the result is no task is visible on page load. Kindly render your tasks array and make sure that your list should appear in the order of array index on UI.

### Requirement
![Screenshot from 2023-08-26 14-21-03](https://github.com/IvonneBenitesRodriguez/todo-list-project/assets/66476329/babf5a0f-ca62-4f3c-8c22-7ddb2bd3ef6d)


### Your UI
![image](https://github.com/IvonneBenitesRodriguez/todo-list-project/assets/66476329/368e801b-b08e-405a-8f57-d412c71e6787)

Reply:
Hello @skyv26 👋🏾
Very nice to meet you
thank you for the suggestions,
 I appreciate it, just wanted to let you know that I already modified it,
 Best regards,